portfolio/src/main/webapp/resume.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/0d47b72ee3.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <nav class="navbar">      
+      <div class="logo">
+        <h3><a href="#">Charles Ogbogu</a></h3>
+      </div>
+      <ul class="nav-links" id="js-menu">
+        <li><a href="index.html" class="navbar-link">About</a></li>
+        <li><a href="resume.html" class="navbar-link">Resume</a></li>
+        <li><a href="#" class="navbar-link">Experience</a></li>
+        <li><a href="#" class="navbar-link">Sport</a></li>
+        <li><a href="#footer" class="navbar-link">Contact</a></li>
+      </ul>
+
+      <span class="navbar-toggle" id="js-navbar-toggle">
+        <i class="fas fa-bars"></i>
+      </span> 
+
+      <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
+      <script>
+        $(document).ready(function() {
+          $('.navbar-toggle').click(function() {
+              $('ul').toggleClass('active');
+          })
+        })
+
+      </script>
```

Comment:
Similar here with the JS. The doc ready and click could also be a re-used file like script.js.